hmi_sdk/HMI_SDK/AppData/AppData.cpp

```diff
@@ -427,21 +371,17 @@ std::vector<SMenuCommand> AppData::getCommandList()
 {
     static std::vector<SMenuCommand> retVec;
     retVec.clear();
-    for(std::vector <SMenuCommand>::iterator iterator = m_vec_scommand.begin(); iterator != m_vec_scommand.end(); iterator++)
-    {
-        if(0 == (*iterator).i_parentID)
-        {
+    for (std::vector <SMenuCommand>::iterator iterator = m_vec_scommand.begin(); iterator != m_vec_scommand.end(); iterator++){
```

Comment:
Please don't make one line code too length. Each line of text in your code should be at most 80 characters long. Thanks.